test task_11

juliamokh opened the pull request:
demo: [link]( https://juliamokh.github.io/assignments_juliamokh/task_11)
source: [link](https://github.com/kottans-frontend-2018/assignments_juliamokh/tree/task_11/task_11)

juliamokh committed:
added task 11

suchov commented:
💣

viktor-yakubiv commented on task_11/index.html:
```diff
@@ -0,0 +1,292 @@
+                        <span>To</span>
+                    </label>
+                    <select id="res-language-1" name="res-language-1">
+                        <option value="arabic" selected>Arabic</option>
```
In general, servers and APIs use [ISO language codes](https://www.loc.gov/standards/iso639-2/php/code_list.php) as actual values, instead of names. It's easy to parse on a server side.

viktor-yakubiv commented:
Code looks great!

viktor-yakubiv commented:
Please, further add a link to preview the page and to the code optionally. It will make easier reviewing process. Thanks a lot!

juliamokh committed:
made ISO language codes as value for <option> tag

juliamokh commented:
@viktor-yakubiv Thanks for feedback. I changed values for <option> tag.

juliamokh committed:
fixed left padding for payment-method-radio

juliamokh committed:
added buttons for trash icons, fixed hover effects

AleshaOleg commented:
Demo not working anymore ☹️

juliamokh committed:
fixed link to demo in readme.md

juliamokh commented:
@AleshaOleg sorry, fixed it, now should work

juliamokh committed:
fixed flex-wrap

juliamokh committed:
fixed trash-icons for safari

juliamokh committed:
added hover effects

juliamokh committed:
fixed layout, added icons for input fields

juliamokh committed:
added pattern for password

turchak commented:
1. In firefox it looks without styles, this [link](https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/) can help you
![screenshot-2018-1-31 task 11 kottans - checkout form](https://user-images.githubusercontent.com/17525239/35648783-89bcaea4-06df-11e8-84f2-260b8cc849d8.png)

2. When i try to click on triangle - nothing happens. Try to solve it with `pointer-events`
![screenshot_56](https://user-images.githubusercontent.com/17525239/35648982-3f4dfde0-06e0-11e8-9d03-d7274d5f420e.png)

3. When i press this button, form try submit. It happens, cause `<button class="trash"></button>` doesn't have attribute `type="button"` for example
![screenshot_57](https://user-images.githubusercontent.com/17525239/35649295-4de5391c-06e1-11e8-96ce-fbff0d14a06c.png)

val-fom commented:
<img width="1206" alt="screen shot 2018-02-01 at 02 07 47" src="https://user-images.githubusercontent.com/17828806/35654331-427c67c6-06f5-11e8-9751-d70c6068a06f.png"> `margin-right: auto` should help

juliamokh committed:
fix:
- added styles for input[type=range] for mos and ms,
- fixed pointer-events for styling icons (<select>),
- added type=button for trash icons,
- fixed width for upload label

juliamokh commented:
@turchak @val-fom 
thanks a lot :+1: , fixed it

turchak commented:
Looks good! 
Two more things)
1. Try to fix it. with `z-index`
![screenshot_58](https://user-images.githubusercontent.com/17525239/35677225-764a6bfe-0757-11e8-8bf6-76e1ca950ef4.png)

2. This about font. You uses only `.woff` font format, but in fonts.min.css you have may other formats which doesn't exist in your folder, cause of it error in console. And i think better don't use `space` in file naming `Mark Simonson - Proxima Nova Alt Regular-webfont.woff` in browser is looks like this `Mark%20Simonson%20-%20Proxima%20Nova%20Alt%20Regular-webfont.woff ` better for me `Mark-Simonson-Proxima-Nova-Alt-Regular-webfont.woff` or `Mark_Simonson-Proxima_Nova_Alt_Regular-webfont.woff`

![screenshot_59](https://user-images.githubusercontent.com/17525239/35677532-a93950e2-0758-11e8-9ef1-24143452adff.png)
![screenshot_60](https://user-images.githubusercontent.com/17525239/35677537-ac33365a-0758-11e8-8566-4547e61c4cf5.png)

juliamokh committed:
fix styles and fonts
- added z-index for input[type=range]-thumb,
- cleaned fonts/

juliamokh commented:
@turchak fixed it, thanks.
I accidentally deleted z-index in the previous step. :confused:
And there was another problem with fonts. In my repo wasn't `Mark Simonson - Proxima Nova Alt Regular-webfont.woff2`. So it caused an error. All other font formats existed. So I deleted `.woff2` from fonts.min.css. And a little cleaned the folder from unnecessary font-families.

turchak reviewed (approved):
It looks all right! 👍
